flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -313,18 +313,12 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
             options.dist = dist
         }
 
-        if let enableAutoNativeBreadcrumbs = arguments["enableAutoNativeBreadcrumbs"] as? Bool,
-           enableAutoNativeBreadcrumbs == false {
-            options.integrations = options.integrations?.filter { (name) -> Bool in
-                name != "SentryAutoBreadcrumbTrackingIntegration"
-            }
+        if let enableAutoNativeBreadcrumbs = arguments["enableAutoNativeBreadcrumbs"] as? Bool {
+            options.enableAutoBreadcrumbTracking = false
```

Comment:
@brustolin can you confirm that `enableAutoBreadcrumbTracking` is the same as filtering `SentryAutoBreadcrumbTrackingIntegration`? the boolean `enableAutoBreadcrumbTracking` didn't exist before.